projects/packages/assets/src/class-assets.php

```diff
@@ -307,12 +319,13 @@ public static function normalize_path( $path ) {
 	 * This wrapper handles all of that.
 	 *
 	 * @since 1.12.0
+	 * @since $$next-version$$ Add a new `strategy` option to leverage >= 6.3. script strategy feature. The `async` option is deprecated.
 	 * @param string $handle      Name of the script. Should be unique across both scripts and styles.
 	 * @param string $path        Minimized script path.
 	 * @param string $relative_to File that `$path` is relative to. Pass `__FILE__`.
 	 * @param array  $options     Additional options:
 	 *  - `asset_path`:       (string|null) `.asset.php` to load. Default is to base it on `$path`.
-	 *  - `async`:            (bool) Set true to register the script as async, like `Assets::enqueue_async_script()`
+	 *  - `async`:            (bool) Set true to register the script as async, like `Assets::enqueue_async_script()`. Deprecated since $$next-version$$ in favor of `strategy`.
```

Comment:
Even though we can't rename the option, we can at least make the docs more correct. 🙂
```suggestion
	 *  - `async`:            (bool) Set true to register the script as deferred, like `Assets::enqueue_async_script()`. Deprecated since $$next-version$$ in favor of `strategy`.
```